homeassistant/components/shelly/entity.py

```diff
@@ -297,17 +298,20 @@ def unique_id(self):
         return f"{self.wrapper.mac}-{self.description.path}"
 
     @property
-    def device_state_attributes(self):
+    def device_state_attributes(self) -> dict:
         """Return the state attributes."""
 
         if self._attributes is None:
             return None
 
-        _description = self._attributes.get("description")
-        _attribute_value = get_rest_value_from_path(
-            self.wrapper.device.status,
-            self.description.device_class,
-            self._attributes.get("path"),
-        )
-
-        return {_description: _attribute_value}
+        attributes = defaultdict(str)
+        for attrib in self._attributes:
+            _description = attrib.get("description")
+            _attribute_value = get_rest_value_from_path(
+                self.wrapper.device.status,
+                self.description.device_class,
+                attrib.get("path"),
+            )
+            attributes[_description] = _attribute_value
```

Comment:
Drop the _ prefix
```suggestion
            description = attrib.get("description")
            attribute_value = get_rest_value_from_path(
                self.wrapper.device.status,
                self.description.device_class,
                attrib.get("path"),
            )
            attributes[description] = attribute_value
```

Reply:
will update code even if it's already merged ;-)